manuscripts/Dockerfile

```diff
@@ -11,12 +11,12 @@ CMD ["R", "-e", "rmarkdown::render('manuscript.Rmd')"]
 ## Non-interactive use:
 ## 
 ##     docker run --name rnexml cboettig/rnexml
-##     docker cp rnexml:/host/manuscript.pdf .
+##     docker cp rnexml:/home/rstudio/rnexml/manuscript.pdf `pwd`
 
 ## Equivalent to:
 ##  
-##     docker run --name rnexml --volumes $(pwd):/host \
-##       --workdir /host --user docker rocker/ropensci \
+##     docker run --name rnexml --volumes $(pwd):/home/rstudio/rnexml \
+##       --workdir /home/rstudio/rnexml --user docker rocker/ropensci \
```

Comment:
It seems to me that `rocker/ropensci` should be changed to `cboettig/rnexml`, but I don't understand enough about Docker to be certain.

Also, the argument `--user docker` results in an error for me (`Error response from daemon: Cannot start container 17223d04169f65e103fadd0b591d3eb6b8657240de7f8953e1863bef740e7edb: finalize namespace setup user get supplementary groups Unable to find user docker`). It works fine if I remove the argument altogether. Is it necessary on some systems? I'm on MacOSX.